src/Features/CSharp/Portable/InlineDeclaration/CSharpInlineDeclarationCodeFixProvider.cs

```diff
@@ -93,35 +92,33 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             if (singleDeclarator)
             {
                 // This was a local statement with a single variable in it.  Just Remove
-                // the entire local declaration statement.  Note that comments belonging to
+                // the entire local declaration statement. Note that comments belonging to
                 // this local statement will be moved to be above the statement containing
                 // the out-var.
                 var localDeclarationStatement = (LocalDeclarationStatementSyntax)declaration.Parent;
                 var block = (BlockSyntax)localDeclarationStatement.Parent;
                 var declarationIndex = block.Statements.IndexOf(localDeclarationStatement);
 
-                bool TryFindPriorSurvivingStatementOnSameLine(out StatementSyntax statement)
+                // Try to find a predecessor Statement on the same line that isn't going to be removed
+                StatementSyntax priorStatementSyntax = null;
+                var localDeclarationToken = localDeclarationStatement.GetFirstToken();
+                for (int i = declarationIndex - 1; i >= 0; i--)
                 {
-                    statement = null;
+                    var statementSyntax = block.Statements[i];
+                    if (declarationsToRemove.Contains(statementSyntax))
+                    {
+                        continue;
+                    }
 
-                    var localDeclarationToken = localDeclarationStatement.GetFirstToken();
-                    for (int i = declarationIndex - 1; i >= 0; i--)
+                    if (!sourceText.AreOnSameLine(statementSyntax.GetLastToken(), localDeclarationToken))
                     {
-                        var statementSyntax = block.Statements[i];
-                        if (declarationsToRemove.Contains(statementSyntax))
-                        {
-                            continue;
-                        }
-                        if (sourceText.AreOnSameLine(statementSyntax.GetLastToken(), localDeclarationToken))
-                        {
-                            statement = statementSyntax;
-                            return true;
-                        }
+                        break;
                     }
-                    return false;
+
+                    priorStatementSyntax = statementSyntax;
```

Comment:
shouldn't we break right after this?  otherwise, we'll keep looking back on the same line, when really we just want the nearest one that will stay.